lesson_28/db/db_app/src/main/resources/queries/amiyahjones.sql

```diff
@@ -0,0 +1,22 @@
+--  returns the counts of media items by type.
+SELECT 
+  type, COUNT(*) AS item_count
+FROM 
+  media_items
+GROUP BY 
+  type;
+
+-- returns the sum of total pages checked out by guests.
+SELECT 
+SUM(pages) AS total_pages
+FROM 
+guests;
+
+-- shows all 5 guests 
+-- and any corresponding records in the checked_out_items table
+SELECT 
+  guests.id , guests.name , ch.item_id , ch.checkout_date
+FROM 
+guests 
+INNER JOIN checked_out_items ch ON 
```

Comment:
lost my train of thought for ON , but I do know i'm gathering information from two tables and need to join them together to reveal the items checked out by each guest

Reply:
Yup, you're on the right track.